aten/src/ATen/native/native_functions.yaml

```diff
@@ -4435,9 +4431,6 @@
 # softmax allows positional dtype, unlike most operators, because kwonly is BC-breaking when loading jit models.
 - func: softmax.int(Tensor self, int dim, ScalarType? dtype=None) -> Tensor
   variants: function, method
-  dispatch:
-    CompositeImplicitAutograd: softmax
-    NestedTensorCPU, NestedTensorCUDA: softmax
```

Comment:
I'm good with this softmax change :) Now that nested tensor can be treated as composite implicit, this properly calls `softmax`, which redispatches to `_softmax`, which has special NT registered kernels for it.